src/states/node.rs

```diff
@@ -720,6 +720,9 @@ impl Node {
                    self,
                    peer_id,
                    hop_msg);
+            // TODO - We could return `UnknownConnection` here and not handle the message, but we
+            //        could be handling a tunnelled message here immediately after the tunnel
+            //        closed.
             *self.name()
```

Comment:
That's not the comment I wanted — I wanted something saying that own name is used in lieu of the peer's name because we no longer have the peer's name, but want to handle the message anyway — and that using own name should be equivalent to using "none" for all uses.